docs(auth): Sign-out example (#8363)
island-is/island.is: handbook/technical-overview/auth/configuration.md

     "amr": ["external"]
   }
   ```

### Example sign-out

1. User is using the site `https://app.myorg.is` which has the `@myorg.is/webapp` client.

> 2. The user clicks a logout button. The client redirects them to the IAS endsession endpoint: `https://innskra.island.is/connect/endsession` with these query parameters:

Reviewer: ```suggestion
2. The user clicks a logout button. The client should locally logout the user before redirecting to the IAS endsession endpoint: `https://innskra.island.is/connect/endsession` with these query parameters:
```

> 

Reviewer: The spec defines more parameters the client could use:
https://openid.net/specs/openid-connect-rpinitiated-1_0.html#RPLogout.
The `state` can be used by the client to pass some state information into the post_logout_redirect_uri page, so I think we should at least add the state parameter to the list as an optional parameter.

   - id_token_hint={YOUR_ID_TOKEN}
   - post_logout_redirect_uri=https://app.myorg.is

> 3. If the post_logout_redirect_uri is valid, IAS redirects the user back to the client using that URI where the client can check if the log out was successful.

Reviewer: ```suggestion
3. If the `post_logout_redirect_uri` is valid IAS redirects the user back to the client using that URI with the `state` as a query parameter.
```
